modules/log6x-loki-sizing.adoc

```diff
@@ -0,0 +1,93 @@
+// Module is included in the following assemblies:
+// * observability/logging/logging-6.0/
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-loki-sizing_{context}"]
+= Loki deployment sizing
+
+Sizing for Loki follows the format of `1x.<size>` where the value `1x` is number of instances and `<size>` specifies performance capabilities.
+
+For logging 6.1+, `1x.pico` configuration defines a single Loki deployment with minimal resource and limit requirements, offering high availability (HA) support for all Loki components. This configuration is suited for deployments that do not require a single replication factor or auto-compaction.
```

Comment:
```suggestion
For logging 6.1+, the `1x.pico` configuration defines a single Loki deployment with minimal resource and limit requirements, offering high availability (HA) support for all Loki components. This configuration is suited for deployments that do not require a single replication factor or auto-compaction.
```

Reply:
If this paragraph does not apply to logging 6.0, I don't think it should be in the 6.0 docs, right?

Reply:
This is into a release branch, so it'll get scooted over to the correct area before merging to main. :)